.github/workflows/e2e.yml

```diff
@@ -126,6 +126,8 @@ jobs:
           distro: ubuntu
         - test-name: replica_set_arbiter
           distro: ubuntu
+        - test-name: replica_set_authentication
+          distro: ubuntu
```

Comment:
👍

Reply:
Ah! I don't see the corresponding changes the workflow files, have you set up the pre-commit hook? (see the end of contributing.md)